lib/octokit/client/projects.rb

```diff
@@ -0,0 +1,314 @@
+module Octokit
+  class Client
+
+    # Methods for Projects API
+    #
+    # @see https://developer.github.com/v3/repos/projects
+    module Projects
+
+      # List projects for a repository
+      #
+      # Requires authenticated client
+      #
+      # @param repo [Integer, String, Repository, Hash] A GitHub repository
+      # @return [Array<Sawyer::Resource>] Repository projects
+      # @see https://developer.github.com/v3/projects/#list-repository-projects 
+      # @example
+      #   @client.projects('octokit/octokit.rb')
+      def projects(repo, options = {})
```

Comment:
I wonder if it would be worthwhile to unify this method with [`org_projects`](https://github.com/octokit/octokit.rb/pull/820/files#diff-9607adc63ef94bd2878d32f654bfca81R52) based on whether or not the `repo` parameter has a slash in it. I'm not sure whether it the convenience would trump the explicitness, though. What do you think?